index.js

```diff
@@ -1,47 +1,56 @@
 /* jshint node: true */
 'use strict';
+var path = require('path');
 var config = require('config');
 
 module.exports = {
     name: 'ember-osf',
+    blueprintsPath: function() {
+	return path.join(__dirname, 'blueprints');
```

Comment:
Minor note, but this should be indented per style guide.

(doing it now avoids noisy cleanup commits later)